chain/cosmos/chain_node.go

```diff
@@ -213,8 +213,7 @@ func (tn *ChainNode) WaitForBlocks(blocks int64) (int64, error) {
 	startingBlock := stat.SyncInfo.LatestBlockHeight
 	mostRecentBlock := startingBlock
 	tn.logger().
-		WithField("initialHeight", startingBlock).
-		Debug("wait for blocks")
+		Info("waiting for blocks", zap.Int64("initialHeight", startingBlock))
```

Comment:
We have logging standards docs at https://github.com/cosmos/relayer/blob/main/docs/logging.md#usage (and identical content in lens) that specifies:

> Log messages should begin with an uppercase letter ...
> Names of log fields should be all lowercase `snake_case` format.

Reply:
TIL. I'll add a section in the README pointing to this style guide or perhaps copy that section into ibctest's README.

Reply:
Fyi, the logging style guide will be another PR.